front/src/modules/activities/timeline/components/TimelineActivity.tsx

```diff
@@ -132,22 +130,7 @@ export function TimelineActivity({ activity }: OwnProps) {
   const body = JSON.parse(activity.body ?? '{}')[0]?.content[0]?.text;
 
   const openActivityRightDrawer = useOpenActivityRightDrawer();
-  const [updateActivityMutation] = useUpdateActivityMutation();
```

Comment:
@emilienchvt I've created a useCompleteTask hook to share this logic. While doing it, I felt that @/activities and @/tasks should be one module actually. This is the same thing.

I would organize maybe
```
activities
- timeline
-- components
-- ...
- task-inbox
-- ...

```
So we can share hooks, queries as much as we can